tools/mapping/gen_mapping_files/runoff_to_ocn/README

```diff
@@ -3,21 +3,38 @@ ABOUT THIS TOOL
 ===============
 
 The src/ directory here contains F90 files and a Makefile to produce the
-runoff_map executable, which reads in a runoff file and an ocean grid file
-and outputs a map from the former to the latter:
-
-1) Computes an initial nearest neighbor mapping from the rof grid cells to
-   active ocean grid cells.
-2) Computes a smoothing matrix on the ocean grid.
-3) The two matrices are multipled together to generate the final mapping file.
+runoff_map executable. The executable reads runoff_map.nml and produces three
+maps:
+
+1) A nearest neighbor mapping from the rof grid to the ocean grid.
+   * The optional file_ocn_coast_mask file allows mapping to a subset of ocean
+     cells, presumably the coastal cells)
+2) A smoothing map from the ocean grid to the ocean grid.
+   * The optional file_ocn_coast_mask file allows mapping from a subset of
+     ocean cells, presumably the coastal cells)
+   * By default, the source domain of this map is only cells that appear as
+     destination cells of the nearest neighbor map. Optionally, one can
+     generate a map that maps from all ocean cells, even if they are not
+     destination cells of the nearest neighbor map. This option allows the
+     smoothing map to be reused when mapping from multiple runoff grids onto
```

Comment:
This description makes it sound like specifying this option is a convenience, and will give you the same final results (from step 3). Is that true? If so, it would help if you stated this explicitly.